Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -3294,6 +3320,13 @@ bool RenderLayerBacking::isUnscaledBitmapOnly() const
     return false;
 }
 
+#if HAVE(HDR_SUPPORT)
+bool RenderLayerBacking::drawsHDRContent() const
```

Comment:
Here I don't know if this is the "only" version or not.

Reply:
This function was renamed `RenderLayerBacking::isReplacedElementWithHDR()`.